lib/theme_settings_manager.rb

```diff
@@ -169,7 +169,7 @@ def is_valid_value?(new_value)
   class Enum < self
     def value
       val = super
-      match = choices.find { |choice| choice == val || choice.to_s == val }
+      match = choices&.find { |choice| choice == val || choice.to_s == val }
```

Comment:
Sorry is there some context for this change? Is it possible to have an enum type setting with no choices? That seems quite odd.

Reply:
This was a workaround for a small issue, but I've replaced it with a proper/clean solution. See https://github.com/discourse/discourse/pull/24071/commits/8f7caef9eb284d4bc4cb9fb6f65acac0ebf45364